package.json

```diff
@@ -48,12 +48,12 @@
         "@testing-library/user-event": "^14.2.5",
         "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1",
         "autoprefixer": "^10.4.7",
-        "babel-jest": "^26.3.0",
-        "babel-loader": "^8.0.6",
+        "babel-jest": "^26.6.3",
```

Comment:
as discussed with @niklasnatter we need to keep babel-jest on ^26 as we are currently stuck with jest ^26 I require the latest version of ^26 here